tcf_website/templates/reviews/review.html

```diff
@@ -12,25 +12,28 @@
                 </a>
             </div>
             <div class="col-xs-12 col-md-10 col-lg-11">
-                <div class="d-flex flex-row align-items-center">
-                    <div>
-                        {% if profile %}
-                        <div class="d-flex">
-                            <div>
-                            <a class="review-header" href="{% url 'course_instructor' course_id=review.course.pk instructor_id=review.instructor.pk %}">
-                                <p><strong>{{ review.course }}</strong> — {{ review.instructor.full_name }}</p>
-                            </a>
-                            </div>
-                            <div class="ml-auto">
-                            <!-- Edit functionality is TBD -->
-                            <!-- <a class="review-edit btn btn-sm btn-primary" href="#">
-                                <i class="fas fa-pencil-alt"></i> Edit
-                            </a> -->
-                            </div>
+                <div class="">
```

Comment:
Do we need ` class=""`?

Reply:
uh not sure why that's there. I'll remove it!